versions.sh

```diff
@@ -30,6 +30,7 @@ for javaVersion in 17 11 8; do
 	# Eclipse Temurin, followed by OpenJDK, and then all other variants alphabetically
 	for vendorVariant in \
 		temurin-focal \
+		temurin-jammy \
```

Comment:
I think we should probably swap these so that `jammy` becomes our default (and use `{}` for consistency with the below):

```suggestion
		temurin-{jammy,focal} \
```

Reply:
Done and done :-)
Just amended the pull-request

Reply:
Are you sure? :grimacing:

(https://github.com/docker-library/tomcat/compare/494458abbdb25d077f9b8f89ba5b10e356523f09..5e0c4e4ab68e25787d2fdd5c27da006f7185a099 doesn't have anything and the proposed change still looks the same :sweat_smile:)